### DISCUSSION TITLE
HM-600 mit und ohne Batterie

### DISCUSSION THREAD

**Discussion by 17tr** [hoylabs/OpenDTU-OnBattery#484] [General]
Hallo zusammen,

ich habe vor am HM-600 
- an einem Kanal direkt ein Solarpanel anzuschließen 
- und am anderen Kanal 2xSolarpannel über Victron 100/20 und Akku 2.5 KW/h 24V Akku

Die Idee ist in etwas so, dass tagsüber der Kanal 1 den Ruhestrombedarf deckt und der Kanal 2 auf 0 begrenzt ist, um die Batterie aufzuladen, um nachts den Ruhestrom aus der Batterie über den Kanal 2 zu decken.

Kann das funktionieren ? Wie muss die Konfiguration dafür aussehen ?
Lassen sich die beiden Kanäle getrennt begrenzen ?
 
VG,
Thomas

**Comment by spcqike** [comment]
Ein das geht nur am wechselrichter nicht. 
 
 Du müsstest den zweiten Eingang physikalisch trennen, bis du ihn benutzen willst. Per Relais, Victron lastausgang oder dergleichen. Allerdings mögen die wechselrichter das nicht auf Dauer (hoher einschaltstrom beim verbinden schadet den Kondensatoren). Da bräuchtest du einen strombegrenzer. 
 
Du kannst dann Tags und nachts nur 300W produzieren. Mehr geht nicht suf einem einzelnen Eingang am HM-600. 


an deiner Stelle würde ich mir wahrscheinlich einfach einen weiteren HM-300/HM-400 holen und den dauerhaft mit der Batterie verbunden lassen und den für nachts nehmen. 

Oder gleich beide Eingänge an die Batterie klemmen und den Victron mit der opendtu Verbinden und Solar passthorugh etc nehmen.

**Comment by 17tr** [comment]
Vielen Dank.

Schade, dass sich die Kanäle nicht getrennt begrenzen lassen. 

Vermutlich werde ich dann doch, wie vorgeschlagen beide Kanäle über die Batterie laufen lassen und mit noch einen Victron MPPT 75/15 kaufen, damit ich das 3. Solarpanel parallel anschließen kann.

**Reply by Tiese** [reply]
Hi 17tr,
das Gleiche hatte ich auch vor und den gleichen Denkfehler gemacht...
...wenn es gehen würde, wäre es dennoch nicht optimal, weil...
- angenommen dein Panel (weiß leider nicht was für welche du hast) erzeugt ca. 315 Watt, dann kommen ca. 300 Watt beim Wechselrichter raus - jetzt werden aber nur 200 Watt verbraucht - würdest du dann sinnloser Weise drosseln (oder auch verschenken) oder doch lieber die 100 Watt in den Akku (wie viel Kapazität der hat ist nicht bekannt) laden?!
- nachts: Wenn dann doch noch ein Verbraucher an ist, wäre es ebenfalls sinnlos, die Energie zu horten, satt an den Verbraucher abzugeben!

**Nur die Energie die eingespeist _UND_ verbraucht wird bringt schließlich den SPAREFFEKT!**
